com.unity.robotics.vhacd/Editor/MeshDecomposerWindow.cs

```diff
@@ -0,0 +1,553 @@
+            m_Settings.AssetsConverted = 0;
+
+            if (m_Settings.TotalAssets > 0)
+            {
+                m_ShowBar = true;
+
+                foreach (var filePath in fileEnumerable)
+                {
+                    var f = filePath.Substring(Application.dataPath.Length - "Assets".Length);
+                    var obj = AssetDatabase.LoadAssetAtPath<GameObject>(f);
+
+                    m_Settings.MeshCountChild = 0;
+                    m_Settings.MeshCountTotal = 0;
+                    yield return EditorCoroutineUtility.StartCoroutine(GenerateConvexMeshes(obj, filePath), this);
+                    Debug.Log($"Generated {m_Settings.MeshCountTotal} meshes on {Path.GetFileName(f)}");
+                }
+            }
+            else
+            {
+                Close();
+            }
+        }
+
+        /// <summary>
+        /// The core function to generate and assign mesh colliders.
+        /// </summary>
+        /// <param name="go">GameObject to convert meshes</param>
+        /// <param name="prefabPath">In Batch Mode; path of file to overwrite</param>
+        IEnumerator GenerateConvexMeshes(GameObject go, string prefabPath = "")
+        {
+            // Instantiate and focus on object in Batch Mode
+            if (m_Settings.GenerationMode == VhacdSettings.Mode.BatchMode)
+            {
+                m_MeshObject = Instantiate(go);
+                Selection.activeObject = m_MeshObject;
+                EditorApplication.ExecuteMenuItem("Edit/Frame Selected");
+                m_MeshObject.name = m_MeshObject.name.Substring(0, m_MeshObject.name.Length - "(Clone)".Length);
+            }
+
+            m_Settings.CurrentFile = m_MeshObject.name;
+            var meshFilters = m_MeshObject.GetComponentsInChildren<MeshFilter>();
+
+            // Generate and assign colliders
+            var meshIndex = 0;
+            foreach (var meshFilter in meshFilters)
+            {
+                var child = meshFilter.gameObject;
+                if (m_Settings.OverwriteMeshComponents)
+                {
+                    var existingColliders = child.GetComponents<MeshCollider>();
+                    if (existingColliders.Length > 0)
+                    {
+                        Debug.Log($"{child.name} had existing colliders; overwriting!");
+                        foreach (var coll in existingColliders) DestroyImmediate(coll);
+                    }
+                }
+
+                var decomposer = ConfigureVhacd(child);
+                yield return new WaitForEndOfFrame();
+                var colliderMeshes = decomposer.GenerateConvexMeshes(meshFilter.sharedMesh);
+                yield return new WaitForEndOfFrame();
+                foreach (var collider in colliderMeshes)
+                {
+                    // Assign and save generated mesh
+                    meshIndex++;
+                    var path = $"{m_Settings.MeshSavePath}/{m_MeshObject.name}/{meshIndex}.asset";
```

Comment:
any where we save stuff i think it would be good to put VHACD in the name so people don't get confused with their own "Meshes" directories
```suggestion
                    var path = $"{m_Settings.MeshSavePath}/VHACD_meshes/{m_MeshObject.name}/{meshIndex}.asset";
```

Reply:
by default, meshes will now save in:

`Assets/<ASSET_NAME>/VHACD/Meshes/TEMP` or `Assets/<ASSET_NAME>/VHACD/Meshes` depending on the mode